.gitignore

```diff
@@ -123,7 +123,8 @@ xcuserdata
 #You want the exact same versions installed in all workspaces, you don't want the "latest version".
 !*.lock
 !Podfile.lock
-#Pods/
+# BaldyAsh's addition not to commit Pods/
```

Comment:
It causes some weird module map errors. So I've decided not to include pods. Problem solved

Reply:
hmm. let's solve the problem with another way. it should work out of the box with pods preinstalled.
Otherwise, we can't guarantee future works if some of the pod will disappear 😱 
So, it's not an incorrect solving of the issue